co2eq/meal.js

```diff
@@ -58,12 +60,14 @@ const CARBON_INTENSITY = { // kgCO2eq / kg
   // dairy
   'Cheese': 8.55, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   'Milk': 1.29, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
+  'Milk (2%)': 1.9, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
   'Cream': 5.64, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   'Milk Yogurt': 1.31, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
+  'Yogurt': 2.2, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
   'Butter': 9.25, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   // vegetables
   'Potatoes': 0.18, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
-  'Vegetables': 2.0, // http://www.greeneatz.com/foods-carbon-footprint.html
+  'Broccoli': 2.0, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
```

Comment:
Let's not replace existing items as this would currently break past meals

Reply:
The original source (http://www.greeneatz.com/foods-carbon-footprint.html) changed some of the names from the source. "Vegetables" is simply incorrect information, the source had that number for Broccoli and greaneatz arbitrarily changed the name.